code/controllers/configuration.dm

```diff
@@ -232,6 +232,8 @@ var/list/gamemode_cache = list()
 	var/dooc_allowed = 1
 	var/dsay_allowed = 1
 
+	var/persistence_enabled = 1

```

Comment:
Shouldn't this be 0 by default, if it is a toggle enable?

Reply:
The idea is it's a toggle DISable. If it's disabled by default, shit won't load at roundstart so enabling it after that would be pointless and just create a new 'blank' save for the round only including stuff from that round (and overwriting the previous 'save files') which then in turn wouldn't be loaded for next time.